packages/workflow/src/sinks.ts

```diff
@@ -44,14 +45,64 @@ export interface SinkCall {
 }
 
 /**
- * Sink interface for forwarding logs from the Workflow sandbox to the Worker
+ * Get a reference to Sinks for exporting data out of the Workflow.
+ *
```

Comment:
What happens with logging for activities, does this affect how activities logging is implemented?

Reply:
No. Sinks is a Workflow only feature. They are required because workflow sandboxing makes it impossible for Workflow code to make direct calls to IO stuff (eg. logging and metrics probably involve newtwork requests and/or file system accesses). There is no such constraint for activity code, so we don't need similar indirections for activities.